sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -900,22 +900,15 @@ case class ShowTablePropertiesCommand(table: TableIdentifier, propertyKey: Optio
   }
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    val catalog = sparkSession.sessionState.catalog
-
-    if (catalog.isTemporaryTable(table)) {
-      Seq.empty[Row]
```

Comment:
We allow users to specify the table properties when creating temporary views.  

```SQL
CREATE TEMPORARY VIEW v1
TBLPROPERTIES(path="3")
AS SELECT 1 AS c1, 2 AS c2
```

Instead of disallowing it, I think we should support it.

Reply:
Sure I will create a PR for this.

Reply:
Upon looking at this further, properties specified in `CREATE TEMP VIEW` is not used in `CreateViewCommand`, so we can revert the behavior back to the original one where we showed empty properties. What do you think?

I will still create a PR to support a permanent view, which is a regression.

Reply:
sounds good to me. please remove the item in the migration guide too. 

Thanks!